full-stack/orders-filters-and-listing.md

```diff
@@ -0,0 +1,128 @@
+# SteelEye full-stack developer - Orders filters and listing problem
+
+The question below is meant to give candidates a sense of the problems we tackle at SteelEye. Please submit solutions in the form of a readme + working code. The problems should take around three hours to complete (2 hrs for the APIs and 1 for the Frontend).
+
+Write an API endpoint that returns a paginated filtered list of Order records adn create a frontend for the same.
+
+## Requirements:
+* API - Python (use one of the Python framework: FastAPI, Flask or Django)
+* Database - Elasticsearch
+* Frontend - React.
+
+## Order record schema:
```

Comment:
This is the _database schema_ of an order. I would state this explicitly as API endpoint payloads also have schemas.

Reply:
Yes, you're right here. I've changed it to the Elasticsearch schema.